components/Hyperlinks.js

```diff
@@ -114,17 +150,35 @@ function Hyperlink({ hyperlink }) {
   );
 }
 
+function PollingHyperlink({ pollingType, pollingId }) {
+  useQuery(POLLING_QUERY[pollingType], {
+    variables: { id: pollingId },
+    pollInterval: 2000,
+  });
+
+  return <CircularProgress />;
+}
+
 /**
- * @param {object[]} props.hyperlinks
+ * @param {object[] | null} props.hyperlinks
+ * @param {'articles'|'replies'?} props.pollingType - poll article or reply for hyperlinks when it's not loaded (null)
+ * @param {string?} props.pollingId - polling article or reply id for hyperlinks when it's not loaded (null)
  */
-function Hyperlinks({ hyperlinks = [] }) {
-  if (!hyperlinks || hyperlinks.length === 0) return null;
+function Hyperlinks({ hyperlinks, pollingType, pollingId }) {
+  if (!((pollingId && pollingType) || (!pollingId && !pollingType))) {
+    throw new Error('pollingType and pollingId must be specified together');
```

Comment:
so this is a xor logic, is it possible to change this condition to:
```
if ( !(pollingId ^ pollingType) ) { ... }
```
?

Reply:
But Javascript don't have logical XOR. JS engines will convert to number and do bitwise XOR instead.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Bitwise_Operators#Bitwise_XOR